apps/aehttp/src/aehttp_dispatch_int.erl

```diff
@@ -411,6 +410,23 @@ handle_request('GetAccountBalance', Req, _Context) ->
             {400, [], #{reason => <<"Invalid account hash">>}}
     end;
           
+handle_request('GetPeers', _Req, _Context) ->
+    case application:get_env(aehttp, enable_debug_endpoints, false) of
+        true ->
+            Peers =
+                lists:map(
+                    fun({URI, LastSeen}) ->
+                        #{uri => URI,
+                          last_seen => LastSeen}
+                    end,
+                    aec_peers:all()),
+            Blocked = aec_peers:blocked(),
+            {200, [], #{peers => Peers,
+                        blocked => Blocked}};
+        false ->
+            {403, [], #{reason => <<"Call not enabled">>}}
```

Comment:
"Call not enabled" sounds a bit strange, but I have no obviously better error message

Reply:
I guess we'd have some other debug calls and they could probably share the same response message.